code/modules/food_and_drinks/recipes/food_mixtures.dm

```diff
@@ -87,6 +87,17 @@
 	for(var/i = 1, i <= created_volume, i++)
 		new /obj/item/reagent_containers/food/snacks/store/cheesewheel(location)
 
+datum/chemical_reaction/fetacheesewheel
+	name = "Feta Cheesewheel"
+	id = "feta_cheesewheel"
+	required_reagents = list("goatmilk" = 40)
+	required_catalysts = list("enzyme" = 5)
+
+/datum/chemical_reaction/fetacheesewheel/on_reaction(datum/reagents/holder, created_volume)
+	var/location = get_turf(holder.my_atom)
+	for(var/i = 1, i <= created_volume, i++)
```

Comment:
the `for(var/i in 1 to X)` format is preferred, as it's easier to read and faster.

Reply:
^

Reply:
I'll make the required changes today.